rb/spec/integration/selenium/webdriver/bidi_spec.rb

```diff
@@ -52,6 +57,32 @@ module WebDriver
           level: BiDi::LogInspector::LOG_LEVEL[:ERROR]
         )
       end
+
+      it 'does not close BiDi session if one window is closed' do
```

Comment:
do we want to hint the position here like the counter test does?

Reply:
The following spec hints at the position of the window `last window` should we / do we need to hint the position here? Does it matter? Just seems inconsistent.

Reply:
Yeah, I don't think this matters.